### PR TITLE
generate rollup source maps

### PR DESCRIPTION
**Pull request by lindner**
This makes debugging runtime code in the server process easier

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "test-extension": "mocha-chrome extension/test/index.test.html",
     "test-mocha-words": "mocha-chrome artifacts/Words/test/index.test.html",
     "test-wdio": "cross-env wdio -b http://${npm_package_config_host}:${npm_package_config_port}/ shell/test/wdio.conf.js",
-    "build:rollup": "rollup -c",
+    "build:rollup": "rollup -c --sourcemap",
```
[the remark this answers is not in the document]
Comment (lindner) [reply package.json]: Odd, this shows up in the pull requests.  The rollup build step has always only been used by the server code.  This option does not add slow down the build.

[the remark this answers is not in the document]
Comment (lindner) [reply package.json]: ping

[the remark this answers is not in the document]
Comment (shaper) [reply package.json]: I think we should consider the server code part of the overall system build, and run related build/tests as part of CI. I'm not sure what our current state around that is, but wanted to make explicit the thought while we're here.

[the remark this answers is not in the document]
Comment (lindner) [reply package.json]: I agree, which is why as of yesterday server is running in ci.  (yay!)

**Pull request merged**